web-admin/src/features/organizations/users/permissions.ts

```diff
@@ -0,0 +1,43 @@
+import type { V1OrganizationPermissions } from "@rilldata/web-admin/client";
+
+/**
+ * @source https://docs.rilldata.com/manage/roles-permissions#organization-level-permissions
+ *
+ * Checks if a user has admin permissions for an organization
+ */
+export function isOrgAdmin(permissions?: V1OrganizationPermissions): boolean {
+  return !!permissions?.admin;
```

Comment:
Just curious - any reason why we have a separate admin and guest prop but not for editor and viewer? @begelundmuller

Reply:
Yeah, I understand it seems a bit weird, but the idea is that `admin` and `guest` in this case are flags with a special internal meaning.

You can imagine in the future, we might introduce more roles, like "owner" and "billing admin", which would also have the `admin=true` despite having a different names.

Reply:
Any chance that we can have roles `editor` and `viewer` in the props? This is so we can have a reliable role source and don't need to maintain a separate org role with the granular permissions for the UI.

Reply:
maybe expose role as a separate field that the UI can use and not rely on these flags ?

Reply:
> Any chance that we can have roles `editor` and `viewer` in the props? 

@lovincyrus, I think the mapping that you're looking for comes from the `adminServiceListRoles` API. It looks like it's the backend's mapping from role->permissions.